src/sbp_fileio.c

```diff
@@ -49,31 +50,33 @@ void sbp_fileio_setup(void)
   );
   static sbp_msg_callbacks_node_t write_node;
   sbp_register_cbk(
-    SBP_MSG_FILEIO_WRITE,
+    SBP_MSG_FILEIO_WRITE_REQUEST,
     &write_cb,
     &write_node
   );
 }
 
 /** File read callback.
- * Responds to a SBP_MSG_FILEIO_READ message.
+ * Responds to a SBP_MSG_FILEIO_READ_REQUEST message.
  *
  * Reads a certain length (up to 255 bytes) from a given offset. Returns the
- * data in a SBP_MSG_FILEIO_READ message where the message length field indicates
- * how many bytes were succesfully read.
+ * data in a SBP_MSG_FILEIO_READ_RESPONSE message where the message length field
+ * indicates how many bytes were succesfully read.
  */
 static void read_cb(u16 sender_id, u8 len, u8 msg[], void* context)
 {
-  (void)sender_id;
   (void)context;
 
+  if (sender_id != 0x42)
```

Comment:
I may have missed this earlier, but sender ID 42 is used by those host computer writing to the flash? This would useful to have defined in a constant (and comment) somewhere in the header.